src/Behat/Context/DefaultCookieManager.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+namespace Metadrop\Behat\Context;
+
+/**
+ * Default cookie manager implementation.
+ *
+ * Manages cookie consent interactions.
+ */
+class DefaultCookieManager implements CookieManagerInterface {
+
+  /**
+   * Selector that accepts default cookie categories.
```

Comment:
"Selector to locate the button accept cookies"

Additionally, why don't we name it '$cookieAcceptSelector'?